src/components/account/ResetPassword.test.tsx

```diff
@@ -33,19 +33,14 @@ describe('ResetPassword', () => {
     const {user, emailField, submitButton, spyOnRequestResetPassword} = setUp()
     const testEmail = 'test@fake.com'
 
-    await waitFor(() => {
-      user.clear(emailField)
-      user.type(emailField, testEmail)
-    })
+    await user.type(emailField, testEmail)
+
```

Comment:
We can now directly `await` user actions, rather than needing to wrap in a `waitFor` to suppress the `act` warnings.